perun-core/src/main/java/cz/metacentrum/perun/core/impl/AttributesManagerImpl.java

```diff
@@ -1064,6 +1064,40 @@ public Object mapRow(ResultSet rs, int i) throws SQLException {
 		}
 	}
 
+	@Override
+	public List<Attribute> getAttributes(PerunSession sess, Host host, List<String> attrNames) {
+		if(!CacheManager.isCacheDisabled()) {
+			List<String> controlledAttrNames = new ArrayList<>();
+
+			for(String attributeName: attrNames) {
+				//check namespace
+				if(attributeName.startsWith(AttributesManager.NS_HOST_ATTR)) controlledAttrNames.add(attributeName);
+			}
+
+			List<Attribute> attrs = perun.getCacheManager().getAttributesByNames(controlledAttrNames, new Holder(host.getId(), Holder.HolderType.HOST), null);
+			return this.setValuesOfAttributes(sess, attrs, host, null);
+		}
```

Comment:
We have recently agreed on removing the cache completely. So please, remove this code as well.

Reply:
Ok, removed.